runtime/rococo/src/lib.rs

```diff
@@ -849,7 +849,18 @@ sp_api::impl_runtime_apis! {
 
 	impl authority_discovery_primitives::AuthorityDiscoveryApi<Block> for Runtime {
 		fn authorities() -> Vec<AuthorityDiscoveryId> {
-			AuthorityDiscovery::authorities()
+			// Past session's authority ids
+			let mut past_authority_ids = runtime_api_impl::get_historical_authority_ids::<Runtime>();
+			// Current and next session's authority ids
+			let mut current_and_next_authority_ids = AuthorityDiscovery::authorities();
```

Comment:
This is slightly wrong. `fn authorities` includes also the current authorities which are implied.